src/main/resources/application.properties

```diff
@@ -11,6 +11,10 @@ ui.options.image.downloadSizes={'Default': 2000}
 ui.options.buttons.about.pdfSinglePage=true
 ui.options.buttons.about.pdfFullDocument=true
 
+# Specify dimensions of IIIF images for social media link previews in map of form {'width': 1200, 'height': 630}
+# If nothing specified, og:image and twitter:image in page metadata will be default thumbnail size instead
+social.options.image.dimensions={'width': 1200, 'height': 630}
```

Comment:
We will need to make sure that this property is set in Puppet before we can roll out these changes.